packages/marketplace/src/components/ui/developer-edition-modal/__tests__/developer-edition-modal.tsx

```diff
@@ -24,14 +26,18 @@ describe('DeveloperEditionModal', () => {
 
   it('should match snapshot when visible', () => {
     const wrapper = mount(
-      <DeveloperEditionModal visible={true} confirmSubscription={jest.fn()} afterClose={jest.fn()} />,
+      <Provider store={store}>
+        <DeveloperEditionModal visible={true} afterClose={jest.fn()} />
+      </Provider>,
     )
     expect(wrapper).toMatchSnapshot()
   })
 
   it('should match snapshot when not visible', () => {
     const wrapper = shallow(
```

Comment:
I think this place should mount if you add provider to cover test for `DeveloperEditionModal`